styles/icons/de/bostrab/hannover/so7.svg

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg width="100%" height="100%" viewBox="0 0 1000 1000" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xml:space="preserve" xmlns:serif="http://www.serif.com/" style="fill-rule:evenodd;clip-rule:evenodd;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:1.5;">
+    <g transform="matrix(5.57714,0,0,4.56075,-300.32,-2092.5)">
+        <rect x="56" y="461" width="175" height="214" style="fill:white;stroke:black;stroke-width:0.19px;"/>
+    </g>
+    <g transform="matrix(0.996633,0,-2.28745e-32,0.844496,-56.633,-14.8969)">
+        <path d="M554.5,155L1000,998L109,998L554.5,155Z" style="fill:rgb(255,28,28);"/>
+    </g>
+</svg>
```

Comment:
Please remove the padding and the black outline. Or does the white rectangle have a black outline in reality?